ext_mongodb.php

```diff
@@ -19,6 +19,108 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+/* == MONITORING ============================================================= */
+namespace MongoDB\Driver\Monitoring;
+
+/* ------ Building blocks for value obejcts */
+abstract class CommandEvent
+{
+	private function __construct()
+	{
+		throw new Exception\RunTimeException("Accessing private constructor");
+	}
+
+	public function getCommandName() : string;
+
+	// do we need this? I don't think so, as you pass in an object
+	public function getContext() : mixed;
+
+	// libmongoc use getHost, but spec says "connectionId".
+	public function getHost() : \MongoDB\Driver\Server;
+
+	// Not adding getHint, as we don't really expose that.
+
+	// technically int64_t. Spec has "Id", and not "ID".
```

Comment:
This agrees with our `Cursor::getId()` method, so no objections on the naming. I believe only the ObjectID class uses all-caps for "ID".

Will we have issues with returning an `int64_t` on 32-bit PHP platforms? Should PHPC return a string?

It's a shame we can't repurpose the CursorId class as a general purpose `int64_t` wrapper. It would be useful for cases such as this.